assets/airports/airportLoadList.json

```diff
@@ -172,6 +172,12 @@
         "name": "Cincinnati-Northern Kentucky International Airport",
         "premium": false
     },
+    {
+        "icao": "kdab",
+        "level": "hard",
```

Comment:
With 25acph, it should be 'easy', although admittedly sequencing C150s together with Citations is quite challenging!